docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -0,0 +1,69 @@
+= Importing code to App Studio
+
+[role="_abstract"]
+After you click **Create an application**, the **Grab some code** page opens. From here, choose one of the following source code options to begin building your application:
+
+* Import your own code.
+* Start with an {ProductName} code sample.
+
+== Importing code from your GitHub repository
+
+To use your own code to build your application, complete the following steps:
+
+. Enter the link to your repository in the **Git repository URL** field. {ProductName} verifies your URL right away. You don’t even have to press **Enter**.
+. Optional: Add a **Git reference** to point to code in a specific branch, tag, or commit that you want to use to build your application.
+. Optional: Indicate a **Context directory** to specify the subdirectory for the source code you want to use. 
+
+[NOTE]
+====
+Make sure this path is correct. You might get an error if your build context folder is your root directory but your Dockerfile is in a subdirectory of that folder.
+====
+[start=4]
+. Click **Import code**. 
```

Comment:
Import code just takes them to the next screen. From there, they can make changes to their app and component configuration, before creating the component and starting a build. I was expecting this procedure to cover a little more, or at least mention that there's more to do.

Reply:
i'm about to push content for configuring/reviewing components. take a look and see what you think is still missing? @mattreid

Reply:
Sorry @jseseCCS, I'm not following how they connect or where I should look. Is there a rebuilt preview? I see the configuration components section above when looking at changed files, but maybe I'm missing how it gets referenced here, during the import process. 

My comment here was trying to say that clicking Import code simply takes them to the next step in the import process, they have to then configure (or simply review) it and click Create application to finish this procedure and land on their newly created app Overview page, so was expecting more steps after 4. Import code.

Reply:
i understand. let me noodle and i'll be back

Reply:
@mattreid i made a minor addition at the end of the topic. is this sufficient? i can't add much more without crossing into the next procedure's content.

Reply:
@jseseCCS Would you consider flipping the order here, so that Code samples are first and then you talk about bringing your own code? The flow here is weird to me - the doc starts by talking about how to bring your own code, and stops that flow to talk about samples, which don't get configured, and then resumes the bring your own code flow in the configuration section at the end. It seems like it would be a more natural flow if you started with samples (the lowest effort and quickest to get started with), and then they can dip their toe deeper by bringing their own code, and have that flow connect better between the description of import and configuring.

Reply:
@mattreid i can, but that would be the opposite of the order that the UI presents. i suppose this matters less, though, in these upstream docs (as opposed to UI help).

flipping!